### PR TITLE
GORA-310: Fix partial field deletion in solr documents

### PR DESCRIPTION
**Pull request by madhawa-gunasekara**

### CODE REVIEW

File: gora-solr-5/src/main/java/org/apache/gora/solr/store/SolrStore.java [hunk 1]
```diff
@@ -17,11 +17,7 @@
 import java.io.IOException;
 import java.net.MalformedURLException;
 import java.nio.ByteBuffer;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Properties;
+import java.util.*;
```
Comment (djkevincr) [inline gora-solr-5/src/main/java/org/apache/gora/solr/store/SolrStore.java]: Shall we do this imports separately?

Comment (lewismc) [inline gora-solr-5/src/main/java/org/apache/gora/solr/store/SolrStore.java]: Yes please

Comment (madhawa-gunasekara) [inline gora-solr-5/src/main/java/org/apache/gora/solr/store/SolrStore.java]: Sure I will modify the pull request. +1

**Comment by madhawa-gunasekara** [comment]
@lewismc @djkevincr 

I have updated the pull request with the suggestions

**Comment by djkevincr** [comment]
+1 from my end, if no objections will go ahead and merge to master :)

**Comment by renato2099** [comment]
👍 looks good @madhawa-gunasekara  thanks all!

**Pull request merged**